docs/tutorial-part-2.rst

```diff
@@ -0,0 +1,977 @@
+   ha-demo-dim-dublin-gw-ip      Microsoft.Network/publicIPAddresses       northeurope
+   ha-demo-dim-dublin-gw         Microsoft.Network/virtualNetworkGateways  northeurope
+   ha-demo-dim-dublin-net        Microsoft.Network/virtualNetworks         northeurope
+   ha-demo-dim-monitor           Microsoft.Compute/virtualMachines         francecentral
+   ha-demo-dim-paris-a           Microsoft.Compute/virtualMachines         francecentral
+   ha-demo-dim-paris-app         Microsoft.Compute/virtualMachines         francecentral
+   ha-demo-dim-paris-b           Microsoft.Compute/virtualMachines         francecentral
+   ha-demo-dim-paris-c           Microsoft.Compute/virtualMachines         francecentral
+   paris-to-amsterdam-vpn        Microsoft.Network/connections             francecentral
+   paris-to-dublin-vpn           Microsoft.Network/connections             francecentral
+   ha-demo-dim-monitorVMNic      Microsoft.Network/networkInterfaces       francecentral
+   ha-demo-dim-paris-appVMNic    Microsoft.Network/networkInterfaces       francecentral
+   ha-demo-dim-paris-aVMNic      Microsoft.Network/networkInterfaces       francecentral
+   ha-demo-dim-paris-bVMNic      Microsoft.Network/networkInterfaces       francecentral
+   ha-demo-dim-paris-cVMNic      Microsoft.Network/networkInterfaces       francecentral
+   ha-demo-dim-paris-nsg         Microsoft.Network/networkSecurityGroups   francecentral
+   ha-demo-dim-monitor-ip        Microsoft.Network/publicIPAddresses       francecentral
+   ha-demo-dim-paris-a-ip        Microsoft.Network/publicIPAddresses       francecentral
+   ha-demo-dim-paris-app-ip      Microsoft.Network/publicIPAddresses       francecentral
+   ha-demo-dim-paris-b-ip        Microsoft.Network/publicIPAddresses       francecentral
+   ha-demo-dim-paris-c-ip        Microsoft.Network/publicIPAddresses       francecentral
+   ha-demo-dim-paris-gw-ip       Microsoft.Network/publicIPAddresses       francecentral
+   ha-demo-dim-paris-gw          Microsoft.Network/virtualNetworkGateways  francecentral
+   ha-demo-dim-paris-net         Microsoft.Network/virtualNetworks         francecentral
+
+
+Install the "pg_autoctl" executable
+-----------------------------------
+
+This guide uses Debian Linux, but similar steps will work on other
+distributions. All that differs are the packages and paths. See :ref:`install`.
+
+The pg_auto_failover system is distributed as a single ``pg_autoctl`` binary
+with subcommands to initialize and manage a replicated PostgreSQL service.
+We’ll install the binary with the operating system package manager on all
+nodes. It will help us run and observe PostgreSQL.
+
+.. code-block:: bash
+
+  az vm run-command invoke \
+     --resource-group ha-demo-dim-monitor \
+     --name ha-demo-dim-${node} \
+     --command-id RunShellScript \
+     --scripts \
+        "curl https://install.citusdata.com/community/deb.sh | sudo bash" \
+        "sudo apt-get install -q -y postgresql-common" \
```

Comment:
This command whined quite a bit:

```
debconf: unable to initialize frontend: Dialog
debconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)
debconf: falling back to frontend: Readline
```

It's odd -- I'd think that `-q -y` would suppress that kind of thing.